recipes/onmail/webview.js

```diff
@@ -0,0 +1,23 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+module.exports = Ferdium => {
+  // TODO: If your onMail service has unread messages, uncomment these lines to implement the logic for updating the badges
```

Comment:
since you have implemented the logic, this `TODO` can be removed imo

Reply:
You are right, removing ...